docs/pages/versions/unversioned/sdk/task-manager.md

```diff
@@ -4,24 +4,25 @@ sourceCodeUrl: 'https://github.com/expo/expo/tree/master/packages/expo-task-mana
 ---
 
 import PlatformsSection from '~/components/plugins/PlatformsSection';
+import SnackInline from '~/components/plugins/SnackInline';
 
 **`expo-task-manager`** provides an API that allows you to manage long-running tasks, in particular those tasks that can run while your app is in the background.
 Some features of this module are used by other modules under the hood. Here is a list of Expo modules that use TaskManager:
 
-- [Location](location.md)
-- [BackgroundFetch](background-fetch.md)
+- [Location](location)
```

Comment:
we need to keep .md around to support [remark-validate-links](https://github.com/remarkjs/remark-validate-links) in the `lint-links` script: https://github.com/expo/expo/blob/e028897fb4bfc5f20c03879535657626679b3ee2/docs/package.json#L16

we don't warn on ci for this yet, we need to do some work to fix a bunch of warnings in cases where use do things like link without the .md extension. open to other ideas for how to prevent breaking things when changing headers and moving changes!

Reply:
@brentvatne The idea behind `.md` removal was inspired but future `docs` files rename to `.mdx`. 

As stated in our conversation on Discord, internal links are resolved correctly in many different cases and `lint-links` fails to cover more cases, than this one.

I will revert the `.md` removals for now, but this change probably will need to be applied again in the future.